src/main/java/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -311,7 +309,7 @@ public boolean match(final Territory t) {
     };
   }
 
-  public static Match<Territory> territoryHasInfraFactoryAndIsLand(final PlayerID player) {
+  public static Match<Territory> territoryHasInfraFactoryAndIsLand() {
```

Comment:
This is now the only `Match` factory method in this class that takes zero parameters.  I wasn't sure if the same convention used in `Matches` applies here.  That is, to use a field instead of a method since it doesn't require additional configuration when it is instantiated.  Please advise if it should be converted to a field.